apps/explorer/src/app/components/search/detect-search.spec.ts

```diff
@@ -99,45 +99,61 @@ describe('Detect Search', () => {
           ok: true,
           json: () =>
             Promise.resolve({
-              result: {
-                tx: query,
+              transaction: {
+                hash: query,
               },
             }),
         })
       )
     );
-    const result = await detectTypeByFetching(query, type);
+    const result = await detectTypeByFetching(query);
     expect(fetch).toHaveBeenCalledWith(
-      `${DATA_SOURCES.tendermintUrl}/tx?hash=0x${query}`
+      `${DATA_SOURCES.blockExplorerUrl}/transactions/${toNonHex(query)}`
     );
     expect(result).toBe(type);
   });
 
-  it("detectTypeByFetching should call fetch with non-hex query it's a party", async () => {
-    const query = 'abc';
-    const type = SearchTypes.Party;
-    // @ts-ignore issue related to polyfill
-    fetch.mockImplementation(
-      jest.fn(() =>
-        Promise.resolve({
-          ok: true,
-          json: () =>
-            Promise.resolve({
-              result: {
-                txs: [query],
-              },
-            }),
-        })
-      )
-    );
-    const result = await detectTypeByFetching(query, type);
-    expect(fetch).toHaveBeenCalledWith(
-      `${DATA_SOURCES.tendermintUrl}/tx_search?query="tx.submitter='${query}'"`
-    );
-    expect(result).toBe(type);
-  });
+  // it("detectTypeByFetching should call fetch with non-hex query it's a party", async () => {
```

Comment:
This is commented out as this is a temporary solution until the new api will make some changes to more efficiently query txs by submitter